Palindrome/pallindrome_no.py

```diff
@@ -0,0 +1,35 @@
+#Python program to find whether a number is a pallindrome or not
```

Comment:
Typo in file name.

Reply:
I have fixed the typo, plz review the changes